src/carousel/carousel.component.ts

```diff
@@ -404,6 +405,16 @@ export class IgxCarouselComponent implements OnDestroy {
             }, this.interval);
         }
     }
+
+    @HostListener("keydown.arrowright", ["$event"])
+    public onKeydownArrowRight(event) {
+        this.next();
+    }
+
+    @HostListener("keydown.arrowleft", ["$event"])
+    public onKeydownArrowLeft(event) {
```

Comment:
I think that the event parameter is unnecessary here. Respectively the passage of the argument through the HostListener too.